ganttproject/src/main/java/net/sourceforge/ganttproject/chart/WeekNumbering.kt

```diff
@@ -0,0 +1,55 @@
+package net.sourceforge.ganttproject.chart
+
+import biz.ganttproject.core.time.CalendarFactory
+import com.google.common.base.Function
+import java.time.ZoneOffset
+import java.time.temporal.WeekFields
+import java.util.*
+
+class WeekNumbering {
+
+
+  companion object {
+
+    @JvmStatic
+    val us = Function { date: Date ->
```

Comment:
you can move these objects out of the class to the file level. You can access them in Java using `WeekNumberingKt.getUsNumbering()` (provided that the object is named `usNumbering`)

Reply:
Done